### DISCUSSION TITLE
Copying Table from One Database to Another

### DISCUSSION THREAD

**Discussion by lizferguson5** [pawelsalawa/sqlitestudio#4933] [Q&A]
Hi there, I am using an older version of SQLite Studio wherein I can right click on a tables from one database and copy, then right click on the title of another database and paste it. I can't do this in the more recent versions of SQLite Studio and get the following error. How do I go about easily copying and pasting tables from one database to another? 

Error: "Error while copying data to table UDF_DG_LF: near "Order": syntax error"

**Comment by pawelsalawa** [comment]
Can you show here the DDL of your  table `UDF_DG_LF`?

**Reply by lizferguson5** [reply]
Hi there! Sure thing, it is: 
CREATE TABLE UDF_DGL_Detection_Events (
    Id                 INTEGER    NOT NULL,
    [Order]            INTEGER,
    Type               CHAR (50),
    Title              CHAR (50),
    PostTitle          CHAR (50),
    DbTitle            CHAR (50),
    Length             INTEGER,
    Topic              CHAR (50),
    NMEA_Module        CHAR (50),
    NMEA_String        CHAR (50),
    NMEA_Position      INTEGER,
    Required           BOOLEAN,
    AutoUpdate         INTEGER,
    Autoclear          BOOLEAN,
    ForceGps           BOOLEAN,
    Hint               CHAR (100),
    ADC_Channel        INTEGER,
    ADC_Gain           REAL,
    Analog_Multiply    REAL,
    Analog_Add         REAL,
    Plot               BOOLEAN,
    Height             INTEGER,
    Colour             CHAR (20),
    MinValue           REAL,
    MaxValue           REAL,
    ReadOnly           BOOLEAN,
    Send_Control_Name  CHAR (50),
    Control_on_Subform CHAR (50),
    Get_Control_Data   CHAR (50),
    [Default]          CHAR (50),
    PRIMARY KEY (
        Id
    )
);

**Reply by pawelsalawa** [reply]
Okay, so this is a bug and needs to be fixed. I've created an issue for the same: #4934

**Reply by lizferguson5** [reply]
Thanks so much!! Really appreciate your help with this- let me know if you need an example database.

**Comment by hulinning2** [comment]
Has this issue been resolved?  Thanks

**Reply by pawelsalawa** [reply]
It might have been resolved together with bugfix for #4844 but I haven't tested this particular issue yet. Nevertheless this one here is also tracked and aimed for fix in 3.4.5 by issue #4934, so I will probably verify it (and fix it if it's still broken) soon.

The 3.4.5 version should be released relatively soon. I think not more than a month.

**Reply by pawelsalawa** [reply]
Okay, I've verified and it wasn't fixed yet, but it is now (in 3.4 branch).

**Reply by hulinning2** [reply]
I think I know the problem on my table. One of the columns named as Index and after I rename to something else then it is OK.   Thanks
